src/Styles.jsx

```diff
@@ -1,10 +1,16 @@
 export const ModalStyle = {
   overlay: {
-    position: 'fixed',
-    top: '15%',
-    left: '25%',
-    right: '25%',
-    bottom: '25%',
+    position: 'absolute',
+    top: '0%',
+    left: '0%',
+    right: '0%',
+    bottom: '0%',
   },
-  content: {}
+  content: {
```

Comment:
![screen shot 2017-02-17 at 10 18 13 am](https://cloud.githubusercontent.com/assets/25034502/23073230/6eb950e6-f4fa-11e6-800c-71978bd307e8.png)
It seems to wrap fine when the browser changes sizes.